package.json

```diff
@@ -37,9 +37,6 @@
   "engines": {
     "node": ">=10.13.0"
   },
-  "peerDependencies": {
-    "typescript": "^2.4.1 || ^3.0.0"
-  },
```

Comment:
Why did you remove `peerDependencies`?

Reply:
the current value is exclusive to v4 - is there a specific format you would like to see?